app/controllers/comments_controller.rb

```diff
@@ -2,6 +2,8 @@ class CommentsController < ApplicationController
   before_action :load_movie, :load_comments, only: %i(create destroy)
   before_action :load_comment, only: :destroy
 
+  load_and_authorize_resource
```

Comment:
giải thích chị code này nào
code `load_and_authorize_resource` và `authorize_resource` khác nhau thế nào em

Reply:
`load_and_authorize_resource` nó sẽ tự động load resource vào biến instance và kiểm tra có được thực hiện action không (thông qua hàm `authorize!`), vì nó gọi cả `load_resource` and `authorize_resource`.
`authorize_resource`: Thêm biến instance và tên phương thức vào hàm `authorize!`

Reply:
rứa thì trên em có beforeaction là load_comment rồi, mà line 5 còn dùng `load_and_authorize_resource` thì nó hơi thừa ko, phải load object comment tới 2 lần

Reply:
(y)